.github/workflows/cd.yml

```
@@ -66,12 +66,9 @@ jobs:
             res = await github.rest.repos.createRelease({
               owner: context.repo.owner,
               repo: context.repo.repo,
-              name: '${{ github.ref_name }}',
+              name: '${{ github.ref_name }}-rc',
               tag_name: '${{ github.ref }}',
-              draft: true,
-              body: 'See [CHANGELOG.md](https://github.com/' +
-                     context.repo.owner + '/' + context.repo.repo +
-                    '/blob/${{ github.ref_name }}/docs/CHANGELOG.md) for details.'
+              body: "Release waiting for review...",
```

Minor consistency nit:
```suggestion
              body: 'Release waiting for review...',
```